packages/fast-components-styles-msft/src/carousel/index.ts

```diff
@@ -0,0 +1,144 @@
+import { DesignSystem, withDesignSystemDefaults } from "../design-system";
+import {
+    ComponentStyles,
+    ComponentStyleSheet,
+    CSSRules,
+} from "@microsoft/fast-jss-manager";
+import { applyLocalizedProperty, Direction } from "@microsoft/fast-jss-utilities";
+import { hoverContrast } from "../utilities/colors";
+import { CarouselClassNameContract } from "@microsoft/fast-components-class-name-contracts-msft";
+
+const styles: ComponentStyles<CarouselClassNameContract, DesignSystem> = (
+    config: DesignSystem
+): ComponentStyleSheet<CarouselClassNameContract, DesignSystem> => {
+    const designSystem: DesignSystem = withDesignSystemDefaults(config);
+    const direction: Direction = designSystem.direction;
+
+    function flipperStyles(): CSSRules<{}> {
+        return {
+            position: "absolute",
+            top: "calc(50% - 20px)",
+            zIndex: "100",
+            display: "none",
+        };
+    }
+
+    return {
+        carousel: {
+            position: "relative",
+            width: "fit-content",
```

Comment:
@nalogiudice do we want to use `fit-content` here with the significant lack of support? If this is to support growing to fit the size of the contents, I think we can get the same scenario with a display property of `inline-block`

Reply:
Once I fix the rendering issue that @janechu is helping me with, I will go back to this and come up with a better solution

Reply:
Fixed